common/messages.proto

```diff
@@ -30,6 +30,7 @@ message NetworkDeviceStatsMessage {
     uint64 bytes_out = 3;
     uint64 packets_in = 4;
     uint64 packets_out = 5;
+    optional string socket_id = 6;
```

Comment:
The `StatsMessage` has multiple levels because we were going to do a different message per message type. e.g. prefferably add `SocketTrafficStatsMessage socket_traffic_stats = 2;` to the `oneof StatsMessageType` above.

See discussion https://github.com/borderzero/border0-proto/pull/41#pullrequestreview-2468836460